R/config.R

```diff
@@ -55,7 +55,7 @@ get_conf <- function(project_name) {
 #' @return the local git repo path
 #' @export
 get_git_repo_path <- function(project_name) {
-  conf_path <- paste0("../conf/", project_name, ".yml")
+  conf_path <- paste0("conf/", project_name, ".yml")
```

Comment:
This is not what you want to do in any of the functions below. You are repeating the code over and over again. Why read the conf.yml here when you already have a get_conf? The input of these functions that access the content of a *parsed conf* should be the parsed conf itself. 

In the notebook, the user would then use get_conf("path/to/conf/project.yml"), get a conf variable which is the parsed yml, and then supply to these functions. 

The notebooks would not need to be edited other than the path to the config file, since the functions in conf.R hide the logic of how to access the elements. The notebook only needs to be changed if the path to the data is different, which makes sense to be at notebook level.